collaboration/communication.md

```diff
@@ -1,54 +1,130 @@
-<!--
-    this template is for inspiration, feel free to change it however you like!
+# Communication
 
-    Careful! be sure to protect your privacy when filling out this document
-        everything you write here will be public
-        so share only what you are comfortable sharing online
-        you can share the rest in confidence with you group by another channel
--->
+## Communication Schedule
 
-# Communication
```

Comment:
Guys, great work! Thank you a lot for your effort. I am thinking of adding 1 additional topic.

Let's specify how we will communicate in case an urgent video call is needed. I suggest this:

In case an urgent video call is needed, we:

- Notify the team via Slack with a brief explanation of the topic and urgency
- Propose at least two time options that accommodate different time zones
- Confirm availability from most participants before scheduling the call